feedback-module/src/lib/hooks/useForm.js

```diff
@@ -12,10 +12,18 @@ export default function useForm() {
 
   const setFeedbackType = (text, feedbackID) => {
     let updatedData = formData;
-    updatedData.feedbackType = {
-      label: text,
-      feedbackID: feedbackID,
-    };
+    if (text && feedbackID) {
+      updatedData.feedbackType = {
+        label: text,
+        feedbackID: feedbackID,
+      };
+    }
+    setFormData(updatedData);
+  };
+
+  const setInteraction = (isInteraction) => {
```

Comment:
Couldn't we theoretically combine these two functions because they are basically doing the same thing? And just pass in a boolean about whether its an interaction or not?

Reply:
Especially since we are calling both these functions every time - it might simplify it